backend/qemu.pm

```diff
@@ -490,6 +490,7 @@ sub start_qemu {
     bmwqemu::save_vars();                 # update variables
 
     mkpath($basedir);
+    runcmd('/usr/bin/chattr', '+C', $basedir);
```

Comment:
I guess better use ioctl directly to set the attribute - it's easier to ignore errors from it :)